### DISCUSSION TITLE
인증 오류 관련 문의

### DISCUSSION THREAD

**Discussion by hoksi** [commerce-api-naver/commerce-solutions#75] [묻고 답하기]
내솔루션 > 솔루션등록하기를 통하여 테스트솔루션 등록후 TYPE을 SELLER로 하여 인증 요청시 오류가 발생합니다.

code : DataAccessDenied
message : 데이터에 대한 접근 권한이 없습니다.

어디를 확인하여야 할까요?

**Comment by commerce-api-naver** [comment]
@hoksi 

> [!IMPORTANT]
> 게시해주신 내용은 커머스솔루션 개발사 연동 관련 문의로 판단되어 관리자가 게시 공간을 이동하였습니다.
> - 최초 게시된 공간: 커머스API 내 스토어 애플리케이션 기술지원 공간 (`/commerce-api`)
> - 이동된 공간: 커머스솔루션 개발사 기술지원 공간 (`/commerce-solutions`)

안녕하세요 커머스API 권현철입니다.

말씀하신 내용에 대해 답변 드리겠습니다.

'인증 토큰 발급 요청API' 호출 시 인증 토큰 발급 타입(`type`)을 `SELLER`로 설정하신경우 `account_id`에 해당하는 판매자가 솔루션을 구독해야 정상 호출이 가능합니다. 실제 해당 판매자가 구독 중이신지 확인 부탁드리며 구독자가 아닌 솔루션 개발사 자기 자신(`"type:SELF"`)에 대한 인증 토큰 발급을 진행을 원하신다면 인증 토큰의 발급 유형(`type` 파라미터)를 `SELF`로 변경하시어 API재호출 시도 부탁드립니다.

감사합니다.

**Comment by hoksi** [comment]
확인하였습니다.
감사합니다.
